axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
+
+
+def load_file(filename, directory):
```

Comment:
Fine by me.

I'm not entirely sure use dfs would make things simpler in this case, the data as is would need to be pivoted for the df to be advantageous or the data could be stored with rows corresponding to "genes" and columns to different strategies... Perhaps not a bad idea (but I don't think necessary for this PR).